cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -559,9 +563,8 @@ def addons_ready():
         return False
 
 
-@when('loadbalancer.available', 'certificates.ca.available',
-      'certificates.client.cert.available')
-def loadbalancer_kubeconfig(loadbalancer, ca, client):
+@when('loadbalancer.available', 'certificates.ca.available')
```

Comment:
Actually, would it be better to wait for `tls_client.ca.saved` here instead? I think would give us a stronger guarantee that the ca file actually exists before we try to generate config.